src/functions.cpp

```diff
@@ -6,6 +6,9 @@
 #include "constants.hpp"
 #include "to_string.hpp"
 #include "inspect.hpp"
+#ifdef _WIN32
```

Comment:
`#if defined(_MSC_VER) && _MSC_VER < 1900` (because it compiles with MinGW and VS2015+ on Windows), and I will move this block before or after `#ifdef __MINGW32__` block (see few lines below in this file).